### DISCUSSION TITLE
Copilot chat - error 400 - The response was filtered due to the prompt triggering Azure OpenAI’s content management policy

### DISCUSSION THREAD

**Discussion by b34rduck** [community/community#58084] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Almost all prompts are now triggering the error 400 - for example typing "what is ?? in js" gives the response:

`unhandled status from server: 400 {"error":{"code":"content_filter","message":"The response was filtered due to the prompt triggering Azure OpenAI’s content management policy. Please modify your prompt and retry. To learn more about our content filtering policies please read our documentation: https://go.microsoft.com/fwlink/?linkid=2198766","param":"prompt","status":400,"type":null}}`

Switching from nightly to stable version doesn't fix it, along with restarting VS Code and making sure it has no updates.

**Comment by Akash1134** [comment]
Hi @b34rduck ,

This is a known issue with Copilot Chat and our engineers are aware of it. Thanks!
